client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -407,6 +408,24 @@ const Editor = (props: EditorProps): JSX.Element => {
       return accessibility;
     };
 
+    const isTabTrapped = () => !!(store.get('monacoTabTrapped') ?? true);
+
+    const setTabTrapped = (trapped: boolean) => {
+      editor.createContextKey('editorTabMovesFocus', !trapped);
```

Comment:
```suggestion
      // Monaco uses the contextKey 'editorTabMovesFocus' to control how it
      // reacts to the tab key. Setting this to true allows the user to tab
      // outside of the editor. If it is false, tab will act inside the editor
      // (i.e. create spaces).
      editor.createContextKey('editorTabMovesFocus', !trapped);
```
I wasn't sure what sorcery was at play here, so I think a comment will help.